### PR TITLE
Quick fixes to get swapr addition to dex.trades by cryptoleishen ready for merging

### PR DESCRIPTION
**Pull request by masquot**
@cryptoleishen is original author who gets credit for bounty.

Rebase + quick fixes to #1756.

### CODE REVIEW

**Comment by masquot** [comment]
@jeff-dude This one is ready to merge if you ask me. Runs perfectly fine locally including seeds tests and incremental run.
 
@cryptoleishen did a great job 👍

File: models/swapr/ethereum/swapr_ethereum_trades.sql [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+{% set project_start_date = '2020-12-09' %}
+
+WITH dexs AS
+(
+    -- Swapr
+    SELECT
+        t.evt_block_time AS block_time,
+        t.`to` AS taker,
+        '' AS maker,
+        CASE WHEN amount0Out = 0 THEN amount1Out ELSE amount0Out END AS token_bought_amount_raw,
```
Comment (jeff-dude) [inline models/swapr/ethereum/swapr_ethereum_trades.sql:24]: just a comment for sharing past experiences: on the decoded tables, columns such as ```amount0out``` or ```amount1out``` are string, which has led to some edge use cases where equality operators don't compute properly without single quotes around the value, i.e. ```'0'``` but i think when it's just ```=``` then it's fine. usually the ```>``` or ```<``` are the culprits

**Review by jeff-dude — APPROVED** [review]
lgtm 🔥

File: models/swapr/ethereum/swapr_ethereum_trades.sql (hunk 1 above)
Comment (masquot) [reply models/swapr/ethereum/swapr_ethereum_trades.sql:24]: thank you for reminding me @jeff-dude

**Pull request merged**